app/models/planning_application.rb

```diff
@@ -92,6 +93,7 @@ class WithdrawOrCancelError < RuntimeError; end
   scope :with_user, -> { preload(:user) }
   scope :for_user_and_null_users, ->(user_id) { where(user_id: [user_id, nil]) }
   scope :prior_approvals, -> { joins(:application_type).where(application_type: {name: :prior_approval}) }
+  scope :non_pending, -> { where.not(status: "pending") }
```

Comment:
Really tiny point but I wonder if there's a better name for this than `non_pending`? Perhaps something like `accepted`, for example. But I'm not completely certain what name would be clear without being potentially confused with a later step (like approval for example), and it's unlikely ever to be a big deal anyway.

Reply:
Hm, active might not be appropriate for closed cases though.

So accepted might work